cmd/security-bootstrapper/main.go

```diff
@@ -0,0 +1,42 @@
+/*******************************************************************************
+ * Copyright 2020 Intel Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"time"
+
+	"github.com/edgexfoundry/edgex-go/internal/security/bootstrapper"
+)
+
+func main() {
+	// temporarily using a default backend for now
```

Comment:
will change this code template structure into using `go-mod-bootsrap` handlers

Reply:
Not sure a program whose job is to open a socket is worth the code bloat.

Reply:
true, but guess if we don't do that, Lenny may not like it-- inconsistent with other cmd app :-O